assets/onjschemas/map_config.onjschema

```diff
@@ -34,18 +44,31 @@ $MapEvent {
 
 }
 
+
 $DecorationDistributionFunction {
 
     $RandomDistributionFunction {
-        decoration: string,
-        density: float,
-        baseWidth: float,
-        baseHeight: float,
-        scaleMin: float,
-        scaleMax: float,
-        onlyCollidesWithNodes: boolean
+        ...distributionFunctionStandardConfig,
+    }
+
+    $SingleClusterDistributionFunction{
+        ...distributionFunctionStandardConfig,
+        minCenterX?: Float,
+        maxCenterX?: Float,
+        minCenterY?: Float,
+        maxCenterY?: Float,
+        innerRadius: Float,
+        outerRadius: Float,
+        nbrOfInnerPoints: Int,
+        nbrOfOuterPoints: Int,
     }
 
+    $MultiClusterDistributionFunction {
+        ...distributionFunctionStandardConfig,
+        blockSize: float,
+        prob: float,
+        additionalProbIfNei: float,
```

Comment:
Please avoid abbreviating names, I don't know what 'Nei' is supposed to mean

Reply:
nei=Neighbor, but ok